ts/sagas/wallet.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * A saga that manages the Profile.
+ */
+
+import { call, Effect, put, takeLatest } from "redux-saga/effects";
+
+import { WalletAPI } from "../api/wallet/wallet-api";
+import {
+  FETCH_CARDS_REQUEST,
+  FETCH_TRANSACTIONS_REQUEST
+} from "../store/actions/constants";
+import { cardsFetched } from "../store/actions/wallet/cards";
+import { transactionsFetched } from "../store/actions/wallet/transactions";
+import { CreditCard } from "../types/CreditCard";
+import { WalletTransaction } from "../types/wallet";
+
+function* fetchTransactions(): Iterator<Effect> {
+  const transactions: ReadonlyArray<WalletTransaction> = yield call(
+    WalletAPI.getAllTransactions
```

Comment:
`getAllTransactions` should be passed as parameter to this method like [here](https://github.com/teamdigitale/italia-app/blob/master/ts/sagas/messages.ts#L65)